src/FillArrays.jl

```diff
@@ -552,4 +560,17 @@ Base.print_matrix_row(io::IO,
         i::Integer, cols::AbstractVector, sep::AbstractString) =
         axes_print_matrix_row(axes(X), io, X, A, i, cols, sep)
 
+
+"""
+    show(io::IO, x::AbstractFill)
+    show(io::IO, ::MIME"text/plain", x::AbstractFill)
+Display concise description.
+"""
```

Comment:
This doc-string is not needed and will just add noise to `? show`

Reply:
Cut.